ES/ESmy015.xml

```diff
@@ -117,12 +115,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                               አብ፤ ዋህድ፤ ይትባረክ፤ እግዚአብሔር፤ ጸራቅሊቶስ፡ መንፈሰ፡ አብ፡ ወወልድ፡ ወመንፈሰ፡ አብ፡ አ<pb n="36vb"/>ምላክ፡ ውእቱ፡</incipit>
                          </msItem>
 
```

Comment:
if you want to open a new msItem you should first close the previous one, so insert
`</msItem>` here